frontend/src/plugin-loader.tsx

```diff
@@ -97,6 +97,16 @@ class PluginLoader extends Logger {
     );
   }
 
+  public refreshPlugins() {
+    this.log('Refreshing plugin list');
+    this.dismountAll();
+    for (const plugin of this.plugins) {
+      this.unloadPlugin(plugin.name);
+    }
+    this.callServerMethod('refresh_plugins');
```

Comment:
pretty sure this method should be async and you should await callServerMethod.

Reply:
The callServerMethod function does indeed need to be awaited. I'll push a commit to fix that whenever I have a chance.

Reply:
The fix has been pushed